infra/modules/notifications/resources/main.tf

```diff
@@ -0,0 +1,77 @@
+# This module manages an SESv2 email identity.
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+locals {
+  mail_from_domain          = "mail.${var.domain_name}"
+  stripped_domain_name      = replace(var.domain_name, "/[.]$/", "")
+  stripped_mail_from_domain = replace(local.mail_from_domain, "/[.]$/", "")
+  dash_domain               = replace(var.domain_name, ".", "-")
+}
+
+# Verify email sender identity.
+# Docs: https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-email-manage-verify.html
+resource "aws_sesv2_email_identity" "sender" {
+  email_identity         = local.stripped_domain_name
+  configuration_set_name = aws_sesv2_configuration_set.email.configuration_set_name
+}
+
+# The configuration set applied to messages that is sent through this email channel.
+resource "aws_sesv2_configuration_set" "email" {
```

Comment:
I see you attach this configuration set as the default configuration set for the email identity, and also set this same configuration set as the default configuration set for the email channel you create in notifications-app. I think we should only do one of the two, not both as it's redundant. Perhaps we could discuss which would be more appropriate to do.

My initial thoughts are that I lean towards starting with the simplest option:
1. Set the default configuration set for the email identity as you already do in this file
2. Don't set any default configuration set for the email channel in modules/notifications-app/resources/main.tf

Once we do that we can simplify the interfaces a bit too and remove the configuration set from the outputs of the module/notifications module (the module that creates the email identity) and also remove it from the variables of the notifications-app module.

Reply:
I'll see if that works!

Reply:
It works!